src/gui/mzroll/forms/alignmentdialog.ui

```diff
@@ -324,7 +324,7 @@
         </item>
         <item>
          <property name="text">
-          <string>Loess Fit</string>
+          <string>Loess Fit  [Deprecated]</string>
```

Comment:
Would suggest moving Loess Fit to be the third option. Also, Either change the "OBI-Warp (beta)" to "OBI-Warp [beta]" or change this to "(deprecated)".

Reply:
>Would suggest moving Loess Fit to be the third option.

Yes, makes sense

>Also, Either change the "OBI-Warp (beta)" to "OBI-Warp [beta]" or change this to "(deprecated)".

Sure

Reply:
Done